app/assets/javascripts/channels/notifications.js

```diff
@@ -15,6 +15,13 @@ const Notifications = {
     } else {
         $(document.body).prepend(notification_html);
     }
+  },
+  refreshClientList: function() {
+    // update the client list
+    $.ajax({
+      type: "GET",
+      dataType: "script"
```

Comment:
I'm trying this out to put [SJR](https://signalvnoise.com/posts/3697-server-generated-javascript-responses) into practice, but it may be a bad test case. I don't really see a great logical advantage of doing it this way in this case.